concerned-coyotes/earlyinternet/news/tests.py

```diff
@@ -0,0 +1,79 @@
+import datetime
+import random
+
+from django.test import TestCase
+from django.utils.dateparse import parse_datetime
+
+from .models import Article
+
+
+class ArticleTestCase(TestCase):
+
+    def setUp(self) -> None:
+        self.article = Article.objects.create(
+            source="HackerNews",
+            author="Guido van Rossum",
+            title="Why Python is such a nice language",
+            description="...",
+            content="...",
+            url="http://python.org/",
+            published_at=datetime.datetime(2020, 1, 1, 12, 0)
+        )
+
+    def test_representation(self):
+        """ Test if Article.__str__ works correctly """
+        self.assertEqual(
+            str(self.article),
+            "Why Python is such a nice language 2020-01-01T12:00:00"
+        )
+
+    def test_article_manager_create_article(self):
+        """
```

Comment:
Typically, you write one-line docstrings for unittest test methods as a single-line docstring will be included in the output of failing tests when you run them. However, this seems to be an accidental insertion by a docstring "helper" in an IDE.